### PR TITLE
Validation service

### PR DESCRIPTION
**Pull request by zjalicf**

### CODE REVIEW

File: validation/src/main/java/com/smartbank/validation/Config/KafkaTopicConfig.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.kafka.config.TopicBuilder;
+
+@Configuration
+public class KafkaTopicConfig {
+
+    @Value("${account.topic}")
+    private String accountTopic;
+
+    @Value("${transaction_request.topic}")
+    private String transaction_requestTopic;
```
Comment (NinaMNina) [inline validation/src/main/java/com/smartbank/validation/Config/KafkaTopicConfig.java]: Naming convention is not good -> https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
Same for the one below

File: validation/src/main/java/com/smartbank/validation/KafkaListeners.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.config.KafkaListenerEndpointRegistry;
+import org.springframework.stereotype.Component;
+
+@Component
+public class KafkaListeners {
+
+    @SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
+    @Autowired
+    KafkaListenerEndpointRegistry registry;
```
Comment (NinaMNina) [inline validation/src/main/java/com/smartbank/validation/KafkaListeners.java]: Add Controlling Access level. Same below

File: validation/src/main/java/com/smartbank/validation/Service/Impl/ValidationServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Service
+public class ValidationServiceImpl implements ValidationService {
+
+    private static Logger LOGGER = Logger.getLogger(String.valueOf(ValidationServiceImpl.class));
+
+    @Autowired
+    SaldoRepository saldoRepository;
```
Comment (NinaMNina) [inline validation/src/main/java/com/smartbank/validation/Service/Impl/ValidationServiceImpl.java]: Add Controlling Access level. Same below

**Review by NinaMNina — CHANGES_REQUESTED** [review]

File: validation/src/main/java/com/smartbank/validation/Config/KafkaTopicConfig.java (hunk 1 above)
Comment (zjalicf) [reply validation/src/main/java/com/smartbank/validation/Config/KafkaTopicConfig.java]: done

File: validation/src/main/java/com/smartbank/validation/KafkaListeners.java (hunk 2 above)
Comment (zjalicf) [reply validation/src/main/java/com/smartbank/validation/KafkaListeners.java]: done

File: validation/src/main/java/com/smartbank/validation/Service/Impl/ValidationServiceImpl.java (hunk 3 above)
Comment (zjalicf) [reply validation/src/main/java/com/smartbank/validation/Service/Impl/ValidationServiceImpl.java]: done

**Review by NinaMNina — APPROVED** [review]

**Pull request merged**